doc/tutorials/nirspec_high_howto.rst

```diff
@@ -0,0 +1,814 @@
+
+.. include:: ../include/links.rst
+
+.. _nirspec_high_howto:
+
+==================================
+Keck/NIRSPEC High Resolution HOWTO
+==================================
+
+Overview
+========
+
+This tutorial is written to serve as both and guide and documentation for reducing 
```

Comment:
Note, typo: `serve as both *a* guide and documentation for reducing `

Reply:
corrected